openapi/cineast/README.md

```diff
@@ -1,4 +1,4 @@
-## @cineast-openapi/api@v1-SNAPSHOT.202110151041
```

Comment:
I guess at some point we could treat the bindings as stalbe (and thus remove the `-SNAPSHOT` suffix)